spec/fixtures/solr_cores/uri_service_solr6/conf/schema.xml

```diff
@@ -64,22 +68,15 @@
   <field name="text" type="text" indexed="true" stored="false" multiValued="true"/>
 
   <!-- suggest copyFields -->
-  <copyField source="value" dest="value_suggest" />
-  <copyField source="value" dest="value_suggest_edge" />
-  <copyField source="value" dest="value_suggest_ngram" />
+  <copyField source="pref_label" dest="pref_label_suggest" />
+  <copyField source="pref_label" dest="pref_label_suggest_edge" />
+  <copyField source="pref_label" dest="pref_label_suggest_ngram" />
+  <copyField source="alt_label" dest="alt_label_suggest" />
+  <copyField source="alt_label" dest="alt_label_suggest_edge" />
+  <copyField source="alt_label" dest="alt_label_suggest_ngram" />
   <!-- sort copyFields -->
-  <copyField source="value" dest="value_ssort" />
-
-
-
-
-
-
-
-
-
-
-
+  <copyField source="pref_label" dest="pref_label_ssort" />
+  <copyField source="alt_label" dest="alt_label_ssort" />
```

Comment:
It might not make sense to be able to sort by alt label, since it's multivalued.  I'm actually not sure how that would work in solr.  We may want to remove this copyField.

Reply:
Oh right, you can't sort by a multivalued field. Will remove.